### PR TITLE
fix(ext/web): remove `ErrorEventInit`'s error default

### LINKED ISSUE #14791: Remove ErrorEventInit's error default
See https://github.com/whatwg/html/pull/7983 and https://github.com/web-platform-tests/wpt/pull/34297.

### PR DESCRIPTION
**Pull request by mrkldshv**
Update WPT and remove `ErrorEventInit`'s error default.
Closes #14791.

### CODE REVIEW

File: ext/web/02_event.js [hunk 1]
```diff
@@ -1082,7 +1082,6 @@
         filename = "",
         lineno = 0,
         colno = 0,
-        error = null,
```
Comment (lucacasonato) [inline ext/web/02_event.js:1085]: `error` still exists, it just does not have a default anymore. Update this line to:

```diff
-        error = null,
+        error,
```

Comment (mrkldshv) [reply ext/web/02_event.js:1085]: Sure, updating tests currently.

Comment (mrkldshv) [reply ext/web/02_event.js:1085]: It should be fixed now.

**Comment by bartlomieju** [comment]
@lucacasonato please take a look; should we upgrade `wpt` submodule in a separate PR before landing this one?

**Review by bartlomieju — APPROVED** [review]
Thanks @mrkldshv!

**Pull request merged**